stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -83,15 +83,29 @@ extension ${Self} : CustomStringConvertible {
   /// A textual representation of the value.
   @_inlineable // FIXME(sil-serialize-all)
```

Comment:
Can anyone explain why this is `@_inlineable`?  This seems to me like a natural ABI boundary, so I'm curious if there was a specific reason for pushing it further down.

Reply:
@moiseev  ?

Reply:
Based on the FIXME, Max added this mechanically, to smoothen the transition from previous versions where everything in the stdlib was implicitly inlinable. Attributes marked like this were added in bulk and they can (and should) be removed when we know better.

Reply:
I'll do some perf testing to see if removing this makes a real difference.